terraform/setup/main.tf

```diff
@@ -11,27 +11,50 @@ resource "aws_dynamodb_table" "performance-dynamodb-table" {
   name           = module.common.performance-dynamodb-table
   read_capacity  = 10
   write_capacity = 10
-  hash_key       = "Year"
-  range_key      = "CommitDate"
+  hash_key       = "Service"
+  range_key      = "CommitHash"
 
   attribute {
-    name = "Year"
+    name = "Service"
+    type = "S"
+  }
+
+  attribute {
+    name = "CommitDate"
     type = "N"
   }
 
   attribute {
-    name = "Hash"
+    name = "CommitHash"
     type = "S"
   }
 
   attribute {
-    name = "CommitDate"
-    type = "N"
+    name = "UseCase"
+    type = "S"
+  }
+
+  global_secondary_index {
```

Comment:
`global_secondary_index` is a second queries that we can query the database besides the primary key. For more information on it, please [following this document ](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html)